engine-src/Game/LambdaHack/Client/UI/InventoryM.hs

```diff
@@ -330,22 +330,22 @@ transition psuit prompt promptGeneric permitMulitple
           in (km, useMultipleDef $ Right km)
         , let km = K.mkChar '!'
           in (km, useMultipleDef $ Left "")  -- alias close to 'g'
-        , let km = revCmd MemberCycle
+        , let km = revCmd $ MemberCycleLevel Forward
```

Comment:
I'm afraid, for consistency we'd need to add the remaining 2 cases here. Bonus points if this can be done by parameterizing the 2 cases already present so that not much more code is added.

Reply:
Okay, done. I put the parameterized versions above. Not sure if that is the best place for it, but it works

Reply:
It's the best place. :)